### ISSUE TITLE
Move over BTextQueryList control from Paladin

### ISSUE THREAD

**Issue by adamfowleruk** [HaikuArchives/LibWalter#7]
A text view with a list view with pluggable query and selected messages

**Comment by CodeforEvolution** [comment]
I’d possibly even ask the Haiku devs on whether or not a control widget like this should be added into the core Haiku system. I feel it would be a very useful class to have.

**Comment by adamfowleruk** [comment]
Yeah I had this discussion on IRC earlier. I figured I'd put brand new things in Paladin to try them out as and when they're needed in that UI. Once they work I can do a bit more QA and put them in LibWalter. This can then be used by app developers without breaking ABI compatibility in the core OS. Or so the theory goes.

**Comment by CodeforEvolution** [comment]
Ok, makes sense.
